### PR TITLE
Removed the redundant scroll in the administration.

### PR DESCRIPTION
**Pull request by deyordanov**
Closes https://github.com/SoftUni-Internal/exam-systems-issues/issues/1521

### CODE REVIEW

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/portals/administration/AdministrationPortal.tsx [hunk 1]
```diff
@@ -534,7 +533,14 @@ const AdministrationPortal = () => {
         <AdministrationThemeProvider mode={currentThemeMode}>
             <CssBaseline />
             <Box>
```
Comment (gogo4ds) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/components/portals/administration/AdministrationPortal.tsx:535]: Do we need the outer Box?

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/portals/administration/AdministrationPortal.tsx [hunk 2]
```diff
@@ -534,7 +533,14 @@ const AdministrationPortal = () => {
...
-                <Box sx={{ display: 'flex', height: '100vh' }}>
+                <Box sx={{
+                    display: 'flex',
+                    overflowY: 'hidden',
+                    '&::-webkit-scrollbar': { display: 'none' },
```
Comment (gogo4ds) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/components/portals/administration/AdministrationPortal.tsx:539]: What if the user selects more rows per page, will it have scroll then?

**Review by gogo4ds — CHANGES_REQUESTED** [review]

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/portals/administration/AdministrationPortal.tsx (hunk 1 above)
Comment (deyordanov) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/components/portals/administration/AdministrationPortal.tsx:535]: Removed it.

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/portals/administration/AdministrationPortal.tsx (hunk 2 above)
Comment (deyordanov) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/components/portals/administration/AdministrationPortal.tsx:539]: There will always be a visible scroll if the data cannot be contained on the screen, there were two scrolls at the same time when zooming in, this has been fixed.

**Review by gogo4ds — APPROVED** [review]

**Pull request merged**